doc/debug/debug_in_vs_code.md

```diff
@@ -19,9 +19,9 @@
 The `launch.json` under `.vscode` folder has already been pre-configured to enable execute `az --help` and break in to debug immediately. You can update it to execute the scenario you need.
 
 1. Set `false` to `stopOnEntry` property to break at the first break point you specified.
-2. Update `args` array to run specified command. You need to make sure the arguments are split by space.
+2. Update the `args` array to run specified command. You need to make sure the arguments are split by space. 
 3. Choose between external termninal and integrated termninal. The latter can be toggle by Ctrl + `
-4. Set `config.python.pythonPath` in either user setting or workspace setting to pick the version of python to be executed.
+4. 
```

Comment:
remove `4.`?

Reply:
Just made the update in the latest commit. Thank you for your feedback on this.